### PR TITLE
Update to use U9

### PR DESCRIPTION
**Pull request by puneet-pdx**
- Support for ArcGIS Runtime SDK for Android 100.9.0
- workaround fix for the Route task issue where the online service fails to compute route if the stops are a mix of 3d and 2d points
- Cleanup usage of deprecated ArcGIS Runtime API

For issue: https://github.com/ArcGIS/open-source-apps/issues/426

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -13,6 +13,7 @@ allprojects {
     repositories {
         jcenter()
         maven { url 'https://esri.bintray.com/arcgis' }
+        maven { url 'http://olympus.esri.com/artifactory/arcgisruntime-repo/' }
```
Comment (eri9600) [inline build.gradle:16]: I think this might confuse external users, we should keep it internal only.

**Review by eri9600 — APPROVED** [review]
@puneet-pdx 
One suggestion, otherwise looks good 👍

**Review by sorenoid — APPROVED** [review]
LGTM, able to download and runs on my machine

**Review by sorenoid — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (puneet-pdx) [reply build.gradle:16]: I put this in on @gunt0001 's suggestion to make it easier for testing with internal builds. We do it already for the toolkit.

https://github.com/Esri/arcgis-runtime-toolkit-android/blob/master/build.gradle#L38

**Pull request merged**